EmergencyRoomQueue.py

```diff
@@ -1,31 +1,111 @@
+import time
 
 
 def main():
+
+    program = True
+
+    #test
+    patient_list_queue = []
+
+    # TEMP CODE
     print("Please do some shit")
+    patient_list_queue.append(Patient(10, "Jacob"))
+    patient_list_queue[0].display_patient()
+    patient_list_queue.append(Patient(2, "Sarah"))
+    patient_list_queue[1].display_patient()
+    # TEMP CODE END
+
+    while(program != False):
+
+        userinput = str(input())
+
+
+        # WINDOW ACTIONS CONDITIONS
+        if(userinput == "Exit"):
+            program = False
+            return 0
+        # Function Call for Sort
+        if(userinput == "Sort"):
+            print("are you sure?")
+            print("yes or no")
+            queueListSort()
 
 #Function that will retrieve patient information and sort it into the linked list
-def insertPatientIntoList():
+def insertPatientIntoList(patient_list_queue, patient_name, patient_prio): #TODO
+    pass
+    print("Patient name: ", patient_name)
+    print("\nPriority: ", patient_prio)
+    update = input("\nUpdate this list? <Y/N>: ")
+    while(update == "y" or update == "Y"):
+        update_patient_name = input("Update patient's name: ")
+        patient_list_queue[patient_name] = update_patient_name
+        update_patient_prior = int(input("Update patient's priority: "))
+        patient_list_queue[patient_prio] = update_patient_prior
+        update = input("\nUpdate this list? <Y/N>")
 
+    print("Update Complete...")
+    return patient_list_queue
 
 #Function that allows patient records to be accessed and manipulated in the list

```

Comment:
This Function call was not meant to have interaction with insertpatientrecord(). The intended use was to let a user search for a specific file within the list, then retrieve that file, allow it to be edited, reinsert it into the list, then resort the whole queue, or section of the queue, at your discretion and ability.